src/public-key.js

```diff
@@ -0,0 +1,40 @@
+const URL = require('url');
+const fetch = require('node-fetch');
+
+const CACHE = {};
+const BASE_URL = 'https://handyman.ambassify.eu/eventbus/';
+const TTL = 300 * 1000;
+
+function getPublicKey(baseUrl = BASE_URL, ttl = TTL, staleOnError = true) {
+    if (CACHE.pendingPromise)
```

Comment:
Not sure if this is copied or not, but using `@ambassify/throttle` there instead of this CACHE seems like a better idea since cache might be used for different `baseUrl`'s.

At the very least this should be `CACHE[baseUrl].pendingPromise`